Dokumentáció/Funkcionális Specifikáció.md

```diff
@@ -2,4 +2,21 @@
 A csapatunk célja egy olyan web alapú számológép fejlesztése, ami elsősorban ingyenes és reklám mentes. Ezen felül pedig rendelkezik minden olyan funkcióval ami ki tudja elégíteni mind egy diák, mind egy informatikai hallgató igényeit is számítások tekintetében, viszont kellőképpen letisztult és felhasználóbarát a felülettel rendelkezik, ahhoz, hogy intuitív legyen a használata. Különböző számolási feladtok megoldását teszi majd lehetővé, legyen az akár sima összeadás, kivonás vagy számrendszerek közötti átváltás. Természetesen nem csak számítógépen lesz elérhető, hanem minél több platformon, tehát tableten és telefonon is. Használatához semmilyen regisztráció nem szükséges, bárki hozzáférhet.
 
 # Jelenlegi helyzet
-Bár rengeteg számológép alkalmazás található a piacon, viszont ezek nagyrésze általában rengeteg zavaró reklámot tartalmaz, vagy esetleg bizonyos funkciók csak fizetés ellenében használhatóak. Akár az is előfordulhat, hogy csak egyetlen platformon érhető el. Pontosan ezekből az okokból kifolyólag döntöttünk úgy, hogy meg alkotunk egy teljesen ingyenes és reklámmentes számológépet, amit akár több platformon is lehet használni.
\ No newline at end of file
+Bár rengeteg számológép alkalmazás található a piacon, viszont ezek nagyrésze általában rengeteg zavaró reklámot tartalmaz, vagy esetleg bizonyos funkciók csak fizetés ellenében használhatóak. Akár az is előfordulhat, hogy csak egyetlen platformon érhető el. Pontosan ezekből az okokból kifolyólag döntöttünk úgy, hogy meg alkotunk egy teljesen ingyenes és reklámmentes számológépet, amit akár több platformon is lehet használni.

```

Comment:
ezt nem tudom miért teszi oda vissza, valami branch csúszás lehet, majd a nagyprojektben perma branchek lesznek, ezzel nem lesz baj, szerintem nem is te csinálod rosszul most se